extensions/BUILD

```diff
@@ -50,18 +50,19 @@ wasm_cc_binary(
 wasm_cc_binary(
     name = "metadata_exchange.wasm",
     srcs = [
+        "//extensions/common:base64.h",
```

Comment:
Do you need this? Since `base64_wasm` is already included as a dep.

Reply:
destroyed.